build.gradle

```diff
@@ -542,27 +545,59 @@ task bundle(type: Zip) {
         rename 'GATKConfig.properties', 'GATKConfig.EXAMPLE.properties'
     }
 
-    from("${buildDir}/${pythonPackageArchiveName}")
-    // When including gatkcondaenv.yml file in the release bundle, strip off the
-    // 'build/' prefix used for the location of the Python package archive.
-    from("scripts/gatkcondaenv.yml", {
-        filter { line -> line.replace("build/${pythonPackageArchiveName}", pythonPackageArchiveName) }
-    })
+    from("$buildDir/$pythonPackageArchiveName")
+    from("$buildDir/$gatkCondaYML")
+    from("$buildDir/$gatkCondaIntelYML")
     into(baseName)
 
     doLast {
         logger.lifecycle("Created GATK distribution in ${destinationDir}/${archiveName}")
     }
 }
 
-task createPythonPackageArchive(type: Zip) {
+// Create the python package archive file, and two GATK conda environment yml files from the template
+// (one for standard GATK and one for running GATK with Intel hardware).
+task condaEnvironmentDefinition() {
+    dependsOn 'pythonPackageArchive'
+    inputs.file gatkCondaTemplate
+    outputs.dir buildDir
 
+    doLast() {
+        def standardCondaBindings = [
+                "condaEnvName":"gatk",
+                "condaEnvDescription" : "Conda environment for GATK Python Tools",
+                "tensorFlowDependency" : "tensorflow==1.4.0"]
+        createFileFromTemplate(gatkCondaTemplate, standardCondaBindings, new File("$buildDir/$gatkCondaYML"))
+
+        def intelCondaBindings = [
+                "condaEnvName":"gatkintel",
```

Comment:
do we want a dash in here?  `gatk-intel`  not sure if that makes sense in the context or not, but it's more readable.